Author: Fatih Aşıcı <fatih@pardus.org.tr>
Description:
 Remove module aliases since the driver might conflict with ethernet drivers and
 cause problems

diff -Naurp broadcom-sta.orig/amd64/src/wl/sys/wl_linux.c broadcom-sta/amd64/src/wl/sys/wl_linux.c
--- broadcom-sta.orig/amd64/src/wl/sys/wl_linux.c	2009-02-09 22:43:32.000000000 +0100
+++ broadcom-sta/amd64/src/wl/sys/wl_linux.c	2009-04-06 00:50:22.000000000 +0200
@@ -258,7 +258,6 @@ static struct pci_device_id wl_id_table[
 	{ PCI_VENDOR_ID_BROADCOM, 0x432d, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0 },
 	{ 0 }
 };
-MODULE_DEVICE_TABLE(pci, wl_id_table);
 
 #ifdef BCMDBG
 static int msglevel = 0xdeadbeef;
diff -Naurp broadcom-sta.orig/i386/src/wl/sys/wl_linux.c broadcom-sta/i386/src/wl/sys/wl_linux.c
--- broadcom-sta.orig/i386/src/wl/sys/wl_linux.c	2009-02-09 22:43:32.000000000 +0100
+++ broadcom-sta/i386/src/wl/sys/wl_linux.c	2009-04-06 00:50:28.000000000 +0200
@@ -258,7 +258,6 @@ static struct pci_device_id wl_id_table[
 	{ PCI_VENDOR_ID_BROADCOM, 0x432d, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0 },
 	{ 0 }
 };
-MODULE_DEVICE_TABLE(pci, wl_id_table);
 
 #ifdef BCMDBG
 static int msglevel = 0xdeadbeef;
